src/pages/en/guides/integrations-guide.md

```diff
@@ -108,6 +108,8 @@ In the future, a helpful `astro add` command will be able to handle all of this
 
 Astro integrations are always added through the `integrations` property in your  `astro.config.mjs` file. 
 
+> Find more information about using and configuring any individual integration by visiting our [integrations page](https://astro.build/integrations) and following the link to its repository on GitHub.
```

Comment:
Not sure what exactly you weren't happy with, but what about something like this?

```suggestion
> Want to know more about a specific integration? Find it in our [integrations directory](https://astro.build/integrations) and follow the link to its repository on GitHub.
```